tests/src/main/java/net/neoforged/neoforge/debug/block/BlockPropertyTests.java

```diff
@@ -22,35 +22,57 @@
 import net.minecraft.world.level.Explosion;
 import net.minecraft.world.level.Level;
 import net.minecraft.world.level.block.Block;
+import net.minecraft.world.level.block.Blocks;
 import net.minecraft.world.level.block.EntityBlock;
 import net.minecraft.world.level.block.entity.BlockEntity;
 import net.minecraft.world.level.block.entity.BlockEntityType;
 import net.minecraft.world.level.block.state.BlockBehaviour;
 import net.minecraft.world.level.block.state.BlockState;
 import net.minecraft.world.level.block.state.StateDefinition;
 import net.minecraft.world.level.block.state.properties.BlockStateProperties;
+import net.minecraft.world.level.levelgen.structure.templatesystem.StructureTemplate;
 import net.minecraft.world.phys.BlockHitResult;
+import net.neoforged.neoforge.common.world.AuxiliaryLightManager;
 import net.neoforged.neoforge.eventtest.internal.TestsMod;
 import net.neoforged.testframework.DynamicTest;
 import net.neoforged.testframework.annotation.ForEachTest;
+import net.neoforged.testframework.annotation.RegisterStructureTemplate;
 import net.neoforged.testframework.annotation.TestHolder;
+import net.neoforged.testframework.gametest.StructureTemplateBuilder;
 import net.neoforged.testframework.registration.RegistrationHelper;
 
 @ForEachTest(groups = BlockTests.GROUP + ".properties")
 public class BlockPropertyTests {
-    @GameTest(template = TestsMod.TEMPLATE_3x3)
+    private static final String TEMPLATE_3x4_BOX = "neotests_level_sensitive_light:light_box_3x3:";
+
+    @RegisterStructureTemplate(TEMPLATE_3x4_BOX)
+    public static final StructureTemplate TEMPLATE3x4 = StructureTemplateBuilder.withSize(3, 4, 3)
+            .fill(0, 0, 0, 2, 3, 2, Blocks.STONE)
+            .set(1, 1, 1, Blocks.AIR.defaultBlockState())
+            .set(1, 2, 1, Blocks.AIR.defaultBlockState())
+            .build();
```

Comment:
Unless you need to reuse the structure in another test, I'd just inline it in the test method (with `registerStructureTemplate`)